build.sbt

```diff
@@ -226,8 +226,9 @@ lazy val V = new {
   def scala2Versions = nonDeprecatedScala2Versions ++ deprecatedScala2Versions
 
   // Scala 3
-  def nonDeprecatedScala3Versions = Seq(scala3, "3.0.0-RC2", "3.0.0-RC1")
-  def deprecatedScala3Versions = Seq("3.0.0-M3", "3.0.0-M2", "3.0.0-M1")
+  def nonDeprecatedScala3Versions =
+    Seq(scala3, "3.0.0-RC3", "3.0.0-RC2", "3.0.0-RC1")
```

Comment:
Those are still used for courses at EPFL, so I will hold off dropping them until end of the semester most likely.